dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/AInprogressItemConverter.java

```diff
@@ -70,11 +70,11 @@ protected void fillFromModel(T obj, R witem, Projection projection) {
         submitter = obj.getSubmitter();
 
         witem.setId(obj.getID());
-        witem.setCollection(collection != null ? converter.toRest(collection, projection) : null);
-        witem.setItem(converter.toRest(item, projection));
-        if (submitter != null) {
-            witem.setSubmitter(converter.toRest(submitter, projection));
-        }
+//        witem.setCollection(collection != null ? converter.toRest(collection, projection) : null);
+//        witem.setItem(converter.toRest(item, projection));
+//        if (submitter != null) {
+//            witem.setSubmitter(converter.toRest(submitter, projection));
```

Comment:
@Atmire-Kristof : Please don't include commented out code in PRs.  This code should either be removed (if no longer needed) or uncommented (if needed).  I'm guessing it should be removed.